energy-integration/governance-v2/src/views.rs

```diff
@@ -44,11 +44,16 @@ pub trait ViewsModule:
     }
 
     fn quorum_reached(&self, proposal_id: ProposalId) -> bool {
-        let total_votes = self.total_votes(proposal_id).get();
-        let total_downvotes = self.total_downvotes(proposal_id).get();
+        let proposal_votes = self.proposal_votes(proposal_id).get();
+        let total_votes = proposal_votes.get_total_votes();
+        let total_up_votes = proposal_votes.up_votes;
+        let total_down_votes = proposal_votes.down_votes;
+        let total_down_veto_votes = proposal_votes.down_veto_votes;
+        let third_total_votes = &total_votes / 3u64;
         let quorum = self.quorum().get();
 
-        &total_votes + &total_downvotes >= quorum && total_votes > total_downvotes
+        
```

Comment:
rename the function to quorum_and_vote_reached as it does 2 things. Quorum check, like if there were at least quorum votes and checks whether the proposal_passed. maybe rename the function to proposal_passed.

Reply:
Done